test/specs/wrapper/contains.spec.js

```diff
@@ -74,13 +74,7 @@ describeWithShallowAndMount('contains', (mountingMethod) => {
     expect(fn).to.throw().with.property('message', message)
   })
 
-  it('returns true when wrapper contains root element', () => {
```

Comment:
this exact tests appeared twice (duplicated by accident I guess) and also description was wrong so updated for this PR (it was a failing test so I noticed it).